library/src/com/google/maps/android/geojson/GeoJsonGeometry.java

```diff
@@ -0,0 +1,14 @@
+package com.google.maps.android.geojson;
+
+/**
+ * Represents a GeoJSON geometry object.
+ */
+public interface GeoJsonGeometry {
+
+    /**
+     * Gets the type of geometry
```

Comment:
Can you include some examples here (or in the implementation classes): "e.g. "LineString", "MultiPoint"" - or mention that it will conform to the GeoJson 'type' specification.